src/shared/dissect-image.c

```diff
@@ -432,7 +435,8 @@ int dissect_image(
                         *ret = TAKE_PTR(m);
 
                         return 0;
-                }
+                } else if (flags & DISSECT_IMAGE_NO_GPT)
```

Comment:
this cannot happen, the first branch already covers this

Reply:
Removed